netdiff/__init__.py

```diff
@@ -3,4 +3,5 @@
 from .parsers.olsr import OlsrParser  # noqa
 from .parsers.batman import BatmanParser  # noqa
 from .parsers.netjson import NetJsonParser  # noqa
+from .parsers.bmx6 import BMX6Parser  # noqa
```

Comment:
Could you rename `BMX6Parser` to `Bmx6Parser` for consistency with the other parsers?